client/signup/steps/free-plans-domain-discount-launch/index.jsx

```diff
@@ -0,0 +1,198 @@
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormLabel from 'calypso/components/forms/form-label';
+
+function handleUpgradeButtonClick( props, selectedDomain, domainUpsellItems ) {
+	const { additionalStepData, stepSectionName, stepName, submitSignupStep, goToNextStep } = props;
+	if ( ! domainUpsellItems ) {
+		return;
+	}
+
+	const step = {
+		stepName,
+		stepSectionName,
+		domainUpsellItems,
+		...additionalStepData,
+	};
+
+	submitSignupStep( step, { domainUpsellItems } );
+	goToNextStep();
+}
+
+function handleSkipButtonClick( props ) {
+	const { additionalStepData, stepSectionName, stepName, submitSignupStep, goToNextStep } = props;
+
+	const step = {
+		stepName,
+		stepSectionName,
+		domainUpsellItems: null,
+		...additionalStepData,
+	};
+
+	submitSignupStep( step, { domainUpsellItems: null } );
+	goToNextStep();
+}
+
+export default function FreePlansDomainDiscountLaunchStep( props ) {
+	const translate = useTranslate();
+	const {
+		flowName,
+		stepName,
+		positionInFlow,
+		signupDependencies: { siteSlug },
+	} = props;
+
+	const subHeaderText = translate(
+		'You can select an easier-to-remember name below at a one-time discounted price:'
+	);
+	const headerText = translate( 'Congrats, %s is almost ready!', {
+		args: [ siteSlug ],
+	} );
+
+	return (
+		<div className="free-plans-domain-discount-launch__step-secton-wrapper">
+			<StepWrapper
+				flowName={ flowName }
+				stepName={ stepName }
+				positionInFlow={ positionInFlow }
+				headerText={ headerText }
+				fallbackHeaderText={ headerText }
+				subHeaderText={ subHeaderText }
+				fallbackSubHeaderText={ subHeaderText }
+				isWideLayout={ false }
+				stepContent={ RecommendedDomains( props ) }
+			/>
+		</div>
+	);
+}
+
+function getDomainName( siteSlug ) {
+	return siteSlug.replace( '.wordpress.com', '.blog' );
+}
+
+function FormattedSuggestion( translate, suggestion, isRecommended ) {
+	const currency = suggestion.currency;
+	return (
+		<div className="free-plans-domain-discount-launch__domain-radio-label">
+			<div className="free-plans-domain-discount-launch__domain-radio-label-domain-wrapper">
+				<div className="free-plans-domain-discount-launch__domain-radio-label-domain">
+					{ suggestion.domain }
+				</div>
+				{ isRecommended && <Badge type="success">{ translate( 'Recommended' ) }</Badge> }
+			</div>
+			<div className="free-plans-domain-discount-launch__registration-fee">
+				<div>
+					{ translate( 'Registration fee: {{strong}}%(originalCost)s{{/strong}}', {
+						args: {
+							originalCost: formatCurrency( suggestion.cost, currency, { stripZeros: true } ),
+						},
+						components: { strike: <strike />, strong: <strong /> },
+					} ) }
+				</div>
+				<small>
+					{ translate( 'Renews at: %(cost)s / year', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Renews at %(cost)s / year' )` *ES Score: 9*